src/shared/util.js

```diff
@@ -1329,6 +1493,64 @@ MessageHandler.prototype = {
     return capability.promise;
   },
   /**
+   * Sends a message to the comObj to invoke the action with the supplied data.
+   * Expect that other side will callback to signal start_complete.
+   * @param {String} actionName Action to call.
+   * @param {JSON} data JSON data to send.
+   * @param {Object} queueingStrategy strategy to signal backpressure based on
+   *                 internal queue.
+   * @param {Array} [transfers] Optional list of transfers/ArrayBuffers.
+   * @return {ReadableStream} ReadableStream to read data in chunks.
```

Comment:
Nit: Let's make this more consistent with how we generally write JSDoc comments (note that the other JSDoc comments in `MessageHandler` aren't entirely correct):
```
* @param {string} actionName - Action to call.
* @param {JSON} data - JSON data to send.
* @param {Object} queueingStrategy - strategy to signal backpressure based on
*   internal queue.
* @param {Array} [transfers] - Optional list of transfers/ArrayBuffers.
* @return {ReadableStream} ReadableStream to read data in chunks.
```